yeat/cli/just_yeat_it.py

```diff
@@ -65,8 +65,21 @@ def options(parser):
     )
 
 
-def fastp_configuration(parser):
-    illumina = parser.add_argument_group("fastp configuration")
+def filter_configuration(parser):
+    illumina = parser.add_argument_group("filter configuration")
+    filter_group = illumina.add_mutually_exclusive_group()
+    filter_group.add_argument(
+        "--skip-filter",
+        dest="skip_filter",
+        action="store_true",
+        help="Skip the filtering step (default)",
+    )
+    filter_group.add_argument(
+        "--no-skip-filter",
+        dest="skip_filter",
+        action="store_false",
+        help="Do not skip the filtering step",
+    )
```

Comment:
We only need a single `store_true` or `store_false` flag here: having both in a mutually exclusive group is redundant.